model/guest/GuestTestTakerSession.php

```diff
@@ -39,6 +39,12 @@ public function getUserLabel()
      */
     public function getInterfaceLanguage()
     {
+        $language = parent::getInterfaceLanguage();
+
+        if (!empty($language)) {
```

Comment:
Can it even be empty?
Isn't the default session implementation returning `DEFAULT_LANG` as a fallback value?

Reply:
Indeed, it can not. I'll need to kinda copy the implementation from the BasicSession to be able to use the `DEFAULT_ANONYMOUS_INTERFACE_LANG` constant here as default :(